services/token_update_nft.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2024 Hedera Hashgraph, LLC
```

Comment:
In general, new files should only show the years for which they were active and not the years in which the entire repository was active.

Reply:
Hey @jsync-swirlds , are you fixing this as part of your documentation work?

Reply:
This can be easily fixed by running spotless with Git history analysis enabled.